package.json

```diff
@@ -41,7 +41,8 @@
     "test:watch": "jest --watch --runInBand",
     "test:debug": "node --inspect node_modules/.bin/jest --runInBand",
     "type-check": "tsc --pretty --noEmit",
-    "watch": "concurrently --raw --kill-others 'yarn relay --watch' 'yarn compile -w' 'yarn emit-types --watch'"
+    "watch": "concurrently --raw --kill-others 'yarn relay --watch' 'yarn compile -w' 'yarn emit-types --watch' 'node scripts/publish-on-change.js'",
+    "integrate": "./scripts/quicklink-to-project.sh"
```

Comment:
Integrate feels a bit weird - maybe `yarn dev force`?

Reply:
That was what I had it as originally. Contextually that's a little weird because some projects us `yarn dev` like others use `yarn start`. It just sounds like starting force in dev mode, heh... which this does, amongst other things. 

Naming is hard.